web/src/upload.js

```diff
@@ -256,15 +256,14 @@ export function setup_upload(config) {
             },
         },
     });
-    uppy.setMeta({
-        csrfmiddlewaretoken: csrf_token,
-    });
```

Comment:
Hmm, removing this CSRF block makes me wonder how we plan for the `tusd` server to authenticate the user.